glidertest/tools.py

```diff
@@ -817,4 +817,178 @@ def plot_ts_histograms(ds: xr.Dataset, ax: plt.Axes = None, **kw: dict) -> tuple
     ax[2].contour(xi, yi, zi, colors='black', alpha=0.5, linewidths=0.5)
     ax[2].clabel(ax[2].contour(xi, yi, zi, colors='black', alpha=0.5, linewidths=0.5), inline=True, fontsize=10)
 
-    return fig, ax
\ No newline at end of file
+    return fig, ax
+
+
+def calc_glider_w_from_depth(ds):
+    """
+    Calculate the vertical velocity of a glider using changes in pressure with time.
+
+    Parameters:
+    pressure (array-like): Array of pressure measurements.

```

Comment:
I think Parameters should be ds here, as in calc_seawater_w

Reply:
Yep, thanks!  Updated and also matched formatting better to the earlier functions.